!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  cluster_max_height: 0.09
  cluster_max_length: 0.25
  cluster_max_size: 20000
  cluster_min_distance_to_polygon: 0.04
  cluster_min_height: 0.011
  cluster_min_size: 40
  cluster_tolerance: 0.03
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      cluster_max_height: 0.09
      cluster_max_length: 0.25
      cluster_max_size: 20000
      cluster_min_distance_to_polygon: 0.04
      cluster_min_height: 0.011
      cluster_min_size: 40
      cluster_tolerance: 0.03
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      name: Default
      normal_radius_search: 0.03
      object_height_above_workspace: 0.052
      outlier_min_neighbors: 5
      outlier_radius_search: 0.03
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      passthrough_filter_field_name: z
      passthrough_filter_limit_max: 0.8
      passthrough_filter_limit_min: 0.0
      prism_max_height: 0.05
      prism_min_height: 0.01
      sac_distance_threshold: 0.01
      sac_eps_angle: 0.09
      sac_max_iterations: 1000
      sac_normal_distance_weight: 0.05
      sac_optimize_coefficients: true
      state: true
      type: ''
      voxel_filter_field_name: z
      voxel_filter_limit_max: 0.2
      voxel_filter_limit_min: -0.15
      voxel_leaf_size: 0.02
    state: []
  normal_radius_search: 0.03
  object_height_above_workspace: 0.052
  outlier_min_neighbors: 5
  outlier_radius_search: 0.03
  passthrough_filter_field_name: z
  passthrough_filter_limit_max: 0.8
  passthrough_filter_limit_min: 0.0
  prism_max_height: 0.05
  prism_min_height: 0.01
  sac_distance_threshold: 0.01
  sac_eps_angle: 0.09
  sac_max_iterations: 1000
  sac_normal_distance_weight: 0.05
  sac_optimize_coefficients: true
  voxel_filter_field_name: z
  voxel_filter_limit_max: 0.2
  voxel_filter_limit_min: -0.15
  voxel_leaf_size: 0.02
state: []
